src/main/java/com/minecolonies/coremod/commands/citizencommands/KillCitizenCommand.java

```diff
@@ -22,7 +23,7 @@
 /**
  * List all colonies.
  */
-public class KillCitizenCommand extends AbstractCitizensCommands
+public class KillCitizenCommand extends AbstractCitizensCommands implements IActionCommand
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "IActionCommand" is implemented by a super class; there is no need to implement it here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1939)